dist/READMEcontainer.md

```diff
@@ -0,0 +1,70 @@
+ovn container construction and operation
+
+NOTES/LIMITS:
+- This only works on a cluster with a single master. Openshift
+  supports multiple masters.
+- Single network cidr only, Openshift supports a list of cidrs.
+- ovs daemon set is expected to be running. Its not part of ovn.
+- Uses node-role.kubernetes.io/master and compute labels. 
+  Eventually there will be ovn specific labels.
+- currently building in ../go_controller and copying files to
+  ../images and put in an image. This will likely change.
+NOTES/LIMITS:
+
+There are two daemonsets that support ovn. ovnkube-master runs
+on the cluster master node, ovnkube runs on the remaining nodes.
+The daemonsets run with hostNetwork: true.
+
+The both daemonsets run the node daemons, ovn-controller and ovn-node.
+In addition the daemonset runs ovn-northd and ovn-master.
+
+The startup sequence requires this startup order:
+- ovs (another daemonset starts this)
```

Comment:
As with above, "(started on the host or via an external daemonset)" or something like that perhaps?

Reply:
ovs will be included in the ovn image